eth/tools/fixture_tests.py

```diff
@@ -641,8 +641,7 @@ def apply_fixture_block_to_chain(block_fixture, chain):
 
     block = rlp.decode(block_fixture['rlp'], sedes=block_class)
 
-    imported_blocks, _ = chain.import_block(block)
-    mined_block = imported_blocks[-1]
```

Comment:
I think I like the new API better anyway, but is `mined_block` ever different from `imported_blocks[-1]`?

Reply:
Yes, if the imported block is not part of the canonical chain then `imported_blocks` will be an empty tuple.

Reply:
Since call-sites should have an exact copy of `mined_block` I considered removing it, but didn't pull the trigger.  Should be easy to do later if we decide to slim the API down.